modules/administration/pages/disconnected-setup.adoc

```diff
@@ -182,6 +183,35 @@ mgr-sync refresh
 mgr-sync list channels
 mgr-sync add channel channel-label
 ----
+////
+
+// NEW: 
+
+.Procedure: Synchronizing a Disconnected Server
+. Install the server container with 
++
+----
+mgradm install podman --mirrorPath /media/disk
```

Comment:
I have a pending PR (https://github.com/uyuni-project/uyuni-tools/pull/324) changing the `--mirrorPath` to `--mirror`. You may want to use directly the new one in this doc.
```suggestion
mgradm install podman --mirror /media/disk
```